### PR TITLE
CHEMH-83 | @jdwjdwjdw | Structure and style News node page

### PR DESCRIPTION
**Pull request by jdwjdwjdw**
# READY FOR REVIEW

# Summary
- See News node page in [figma](https://www.figma.com/file/wq9Y5igro1BLYN0kbeojC7/ChEM-H-Design-02?type=design&node-id=201-281&mode=design&t=RhPnuFwhNv33ctJ9-0)
- [CHEMH-83](https://stanfordits.atlassian.net/browse/CHEMH-83): Style News Node
  - wasn't able to match designs completely due to drupal/subtheme limitations, but got as close as I could.
- [CHEMH-242](https://stanfordits.atlassian.net/browse/CHEMH-242): Weird treatment on load more button post Layout Paragraphs

# Review By (Date)
- When convenient

# Urgency
- Normal

# Review Tasks

## Setup tasks and/or behavior to test

1. Check out this branch
2. Create an example News item, and navigate to the News node page
3. Confirm that the news node page matches figma (outside of limitations with field order/display)
4. Confirm that the `See All News` button at the bottom of the news node page looks correct and fixes the issue mentioned in [CHEMH-242](https://stanfordits.atlassian.net/browse/CHEMH-242)
5. Review code 📰 

# Associated Issues and/or People
- [CHEMH-83](https://stanfordits.atlassian.net/browse/CHEMH-83): Style News Node
- [CHEMH-242](https://stanfordits.atlassian.net/browse/CHEMH-242): Weird treatment on load more button post Layout Paragraphs

[CHEMH-83]: https://stanfordits.atlassian.net/browse/CHEMH-83?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ
[CHEMH-242]: https://stanfordits.atlassian.net/browse/CHEMH-242?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ
[CHEMH-83]: https://stanfordits.atlassian.net/browse/CHEMH-83?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ

### CODE REVIEW

**Review by jenbreese — CHANGES_REQUESTED** [review]
There's a lot of extra space after the more news cards for both the responsive and desktop sizes. 
Screenshot 2023-09-26 at 9 40 14 AM
Screenshot 2023-09-26 at 9 39 54 AM

**Comment by jdwjdwjdw** [comment]
> There's a lot of extra space after the more news cards for both the responsive and desktop sizes. Screenshot 2023-09-26 at 9 40 14 AM Screenshot 2023-09-26 at 9 39 54 AM

Thanks @jenbreese! I fixed that up and addressed some other spacing issues in situations where there isn't a dek / a top banner is/isn't used. Should be good to re-review now.

File: src/scss/components/news/_news-node.scss [hunk 1]
```diff
@@ -1,6 +1,180 @@
...
+      }
+    }
+
+    .section-components {
+      max-width: 100%;
+      width: 100%;
+
+      .main-region {
+        @include grid-media-min('sm') {
+          -webkit-box-flex: 0;
```
Comment (jenbreese) [inline src/scss/components/news/_news-node.scss]: @jdwjdwjdw I'm don't think you need to have the -webkit prefixes.

**Review by jenbreese — CHANGES_REQUESTED** [review]

File: src/scss/components/news/_news-node.scss (hunk 1 above)
Comment (jdwjdwjdw) [reply src/scss/components/news/_news-node.scss]: Updated!

File: src/scss/components/news/_news-node.scss [hunk 2]
```diff
@@ -136,36 +136,26 @@
 
       .main-region {
         @include grid-media-min('sm') {
```
Comment (jenbreese) [inline src/scss/components/news/_news-node.scss:138]: @jdwjdwjdw Can we discuss these media queries? When I'm looking at the results in the browser, only the one on line 138 is in effect, all the later ones are overridden.

**Review by jenbreese — APPROVED** [review]

**Pull request merged**